### ISSUE TITLE
Using touch controls to drop an item destroys it instead 

### ISSUE THREAD

**Issue by beelzebielsk** [diasurgical/devilutionX#4847]
**Important information**
Operating system: PSVita
DevilutionX version: 1.4.0, downloaded release

**Describe**
Using touch controls I can destroy items. If I pick up an item from my inventory and then touch outside the inventory window, the item disappears rather than drops. 

**To Reproduce**
1. Start any game
2. Open your inventory 
3. Select an item from it
4. Touch outside of the inventory menu

**Expected behavior**
I expect the item to drop to the ground.

**Additional context**
Tested in hellfire single-player if that matters.

**Comment by Kiiro-Yakumo** [comment]
If I may, I did some testing on this field as well. I thought that maybe there is some kind of funny bug that items were thrown into the town portal which I had opened nearby. Tried throwing items near the pile, outside of the pile, empty space around me to make sure they are not disappearing because of no space on the ground. Every time dropping items with touch makes them go poof without a sound. A silent disintegration.

I have checked it on the version 1.4.1 on PlayStation Vita.
